sdk/sphere/Azure.ResourceManager.Sphere/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Please specify the desired release date here, like 2023-06-30

Reply:
fixed in latest commit